content/resources/registry/publishing/_index.md

```diff
@@ -0,0 +1,183 @@
+enabling others in the scientific community reference and build upon your work.
+
+1. Go to [Zenodo](https://zenodo.org/login).
+2. Use your GitHub account to log in.
+3. Once you are logged in, choose "Github" from the drop-down menu by clicking the arrow next to your email address
+in the top right-hand side menu. 
+![Picture of Zenodo drop down menu](zenodo-github-page.png)
+4. On this page you will see a list of your repositories. Flip the switch next to the recognizer repository you
+created to ON-position.
+![Picture of Zenodo repository switch](zenodo-repo-switch.png)
+5. Go to Github and [create a release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).
+Zenodo will automatically generate a DOI for this release which you should now see in Zenodo GitHub page.
+Zenodo will generate a new DOI version for each release.
+[Read about Zenodo Versioning](https://help.zenodo.org/#versioning).
+6. Click either the title or the DOI-badge to go to your generated DOI-page. 
+7. Finally, you can copy the DOI-badge from the DOI-page by clicking the DOI-badge on the side panel.
+![Picture of Zenodo DOI-badge in the side panel](zenodo-doi-badge.png)
+We recommend copying the Markdown text and adding it to your README-file in your GitHub Recocnizer Repository.
+
+#### Step 7. Add or modify a citation information file
+
+To make your GitHub recognizer repository easily citable we recommend adding a citation information file (`CITATION.cff`).  We have added a citation file to the recognizer template for your convenience. 
+
+1. Modify the `CITATION.cff` file in your repository.
+![Screenshot of the Citation.cff file.](citation-file-edit.png)
+You can edit files in your browser by clicking the pen icon.
+Replace the author name, the ORCID, the title and the github link fields with your own details.
+2. Alternatively you can use the
+[CITATION.cff generation website](https://citation-file-format.github.io/cff-initializer-javascript/#/) to create the
+`CITATION.cff` file  
+3. Once you are happy with your changes, save the file by clicking
+the "Commit changes..." button. You can commit your changes directly to the main branch. 
+4. If you will be collaborating with multiple authors we recommend [learning more about working with branches in GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches). 
+
+#### Step 8. List your recognizer in the registry
+Please see the [Contribute to the registry](/resources/registry/contribute/) page. 
+
+## FAQ
+
+### What to do when your training data set is too large?
+
+Storing data in a repository is not always the right choice. In each folder where it is relevant you should include:
+
+1. Small sets of audio samples 
+2. A README.md containing
+  - provenance of any data included
```

Comment:
needs four spaces for correct indentation

Reply:
otherwise they line up at the wrong level:
![image](https://github.com/ecoacoustics/website/assets/1452968/1a475b3d-e8d8-4d3c-b10c-666bf4bc9a7c)